scraper.rb

```diff
@@ -16,22 +16,22 @@ def noko_for(url)
 
 def scrape_list(url)
   noko = noko_for(url)
-  noko.css('section#comments table tr a[href*="search"]/@href').map(&:text).uniq.each do |link|
+  noko.css('.table-responsive.table-striped tr a[href*="search"]/@href').map(&:text).uniq.each do |link|
     mp_url = URI.join url, link
     mp = noko_for(mp_url)
-    box = mp.css('section#main')
-
-    data = { 
-      id: mp_url.to_s[/search.php\?id=(.*)/, 1],
-      name: box.css('h2').text.gsub('Hon. ', '').strip,
-      constituency: box.xpath('.//h3[contains(., "District Representative")]/following-sibling::h3[1]').text,
-      party: box.xpath('.//h3[contains(., "Party List")]').text.gsub(/Party List\s*-\s*/, ''),
-      image: box.css('img[src*="images/16th"]/@src').text,
-      term: 16,
-      source: mp_url.to_s,
+
+    data = {
+      id:           mp_url.to_s[/search.php\?id=(.*)/, 1],
+      name:         mp.css('.text-primary').text.gsub('Hon. ', '').strip,
+      constituency: mp.xpath('//*[contains(@class,"text-primary")]/following::small[1][contains(text(),"District Representative")]').text,
+      # party: Not available
```

Comment:
An example of an MP for whom this _is_ available is: http://www.congress.gov.ph/members/search.php?id=abayon-hn